fmriprep/cli/run.py

```diff
@@ -61,11 +63,15 @@ def main():
 
     sentry_sdk = None
     if not config.execution.notrack and not config.execution.debug:
+        import atexit
+
         import sentry_sdk
 
-        from ..utils.sentry import sentry_setup
+        from ..utils.telemetry import sentry_setup, setup_migas
 
         sentry_setup()
+        setup_migas(init_ping=True)
+        atexit.register(_migas_exit)
```

Comment:
```suggestion
        atexit.register(migas_exit)
```

Reply:
@mgxd Went ahead and committed this on my own. If you'd rather it be `_migas_exit` in both locations, that's also fine with me.